recipes/grpc/all/conanfile.py

```diff
@@ -147,8 +142,8 @@ def validate(self):
             )
 
     def build_requirements(self):
-        if not self._is_legacy_one_profile:
-            self.tool_requires("protobuf/<host_version>")
+        self.tool_requires("cmake/[>=3.25 <4")
```

Comment:
```suggestion
        self.tool_requires("cmake/[>=3.25 <4]")
```

Reply:
OMG, thanks for fixing this, I was using a pinned version of cmake, then realized that ConanCenter recipes uses a range and changed it last minute before committing.